core/conversion/conversionctx/ConversionCtx.h

```diff
@@ -11,6 +11,10 @@
 #include <cuda_runtime.h>
 #include "core/util/prelude.h"
 
+#define DLA_SRAM_SIZE 1048576
```

Comment:
Maybe we should put these constants somewhere in utils if they will get used in more than one place

Reply:
I think these values would be used only in the conversion context for building the TRT engine. Hence, I kept it encapsulated in ConversonCtx. Do you think it will be used in converters, evaluators or runtime too?

Reply:
These are user definable values right? Presumably compile spec would use this

Reply:
Yeah, moved them to core/utils